app/src/main/java/ch/epfl/sdp/GoogleMapApi.java

```diff
@@ -136,12 +133,32 @@ synchronized public void displayEntity(Displayable displayable) {
                 break;
             case ENEMY:
                 displayMarkerCircle(displayable, Color.RED, "Enemy", 1000);
+                break;

```

Comment:
I know switch case is easier but I think for more elegance its better to use polymorphism or implementation of Abstract Factory design or enum

Reply:
It's actually a good idea. I will try changing it for next week